server/eclipse-project/src/main/java/au/org/democracydevelopers/corla/model/vote/IRVChoices.java

```diff
@@ -88,16 +88,31 @@ public int getRawChoicesCount() {
   /**
    * Constructor - takes a list of IRVPreferences (ranked candidates) representing a vote, sorts
    * this list by rank (most to least preferred), then stores in an unmodifiable list.
-   *
-   * @param mutableChoices a list of IRVPreferences, which need not be valid - repeats or skipped
-   *                       preferences are allowed.
+   * This private constructor is used only during the application of rules for removing invalid
+   * choices (overvotes, skipped preferences, repeated candidate names).
+   * @param sortedChoices a list of IRVPreferences, which need not be valid - repeats or skipped
```

Comment:
Here, the comment on firstDropped I don't think is correct. If firstDropped is 1, you would get a list of size 1 (ie. sublist will include the item at index 0 but skip from 1 onward).

Reply:
Oops, thanks, now fixed. (If firstDropped is 0, we get an empty list.)